src/build.jl

```diff
@@ -0,0 +1,105 @@
+# This file generated the Markdown documentation files.
+
+# The @file macro generates the documentation for a particular file
+# where the {{method1, methods2}} includes the documentation for each method
+# via the `buildwriter` function.
+
+# Currently this prints the methodtable followed by the docstring.
+
+using LightGraphs
```

Comment:
Ah, this causes the build failure. I think this is something silly... I think I need to do

    using $m

inside the right scope (src/build.jl).